ui/app/AppLayouts/Chat/views/CommunitySettingsView.qml

```diff
@@ -34,6 +34,7 @@ StatusAppTwoPanelLayout {
     property var rootStore
     property var community
     property var chatCommunitySectionModule
+    property bool hasAddedContacts: false
```

Comment:
This property is not set from anywhere (always false) :thinking: Was it intended?

Reply:
@osmaczko no I think I set it :)

Reply:
hmm.. can you point me to the line?

Reply:
Yeah i lost it for commit :D